include/ebpf_program_types.h

```diff
@@ -28,13 +28,19 @@ typedef struct _ebpf_program_type_descriptor
     char is_privileged;
 } ebpf_program_type_descriptor_t;
 
+typedef enum _ebpf_helper_function_flags
+{
+    EBPF_HEPER_FUNCTION_FLAGS_UNWIND_ON_SUCCESS = 1,
+} ebpf_helper_function_flags;
```

Comment:
Unless we use this type somewhere, we can just use `#define` instead of enum for now.
If we do need this type, then types should end in `_t`.